src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -124,7 +124,17 @@ export default defineComponent<Props>({
         const state = reactive({
             widgetConfig: computed(() => (props.widgetConfigId ? getWidgetConfig(props.widgetConfigId) : undefined)),
             widgetOptionsJsonSchema: {} as JsonSchema,
-            inheritableProperties: computed<string[]>(() => state.widgetConfig?.options_schema?.inheritable_properties || []),
+            inheritSchemaProperties: computed(() => {
+                const variablesSchema = dashboardDetailState.variablesSchema;
+                const properties = state.widgetConfig?.options_schema?.schema?.properties ?? {};
+                const schema = {};
+                Object.keys(properties).forEach((propertyName) => {
+                    const propertySchema = properties[propertyName] ?? {};
+                    if (!propertySchema) return;
```

Comment:
Question: Isn't `propertySchema` always `true`?

Reply:
Oh yes. I just added if statement without thinking 😅 thank u~

Reply:
I will fix it after PR~